src/components/popover-container/popover-container.pw.tsx

```diff
@@ -0,0 +1,683 @@
+      mount,
+      page,
+    }) => {
+      await mount(
+        <Box display="flex" justifyContent="center">
+          <PopoverContainerComponent position={position} />
+        </Box>
+      );
+
+      const popoverContainerContentElement = popoverContainerContent(page);
+
+      await expect(popoverContainerContentElement).toHaveCSS(
+        "position",
+        "absolute"
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "top",
+        top
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "right",
+        right
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "bottom",
+        bottom
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "left",
+        left
+      );
+    });
+  });
+
+  ([
+    ["left", 0, 0, -76.5, -348],
+    ["right", 0, -348, -76.5, 0],
+  ] as [
+    PopoverContainerProps["position"],
+    number,
+    number,
+    number,
+    number
+  ][]).forEach(([position, top, right, bottom, left]) => {
+    test(`when position prop set to ${position} and a custom open component is used, expected absolute positioning is applied`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <Box display="flex" justifyContent="center">
+          <PopoverContainerComponent
+            position={position}
+            renderOpenComponent={({
+              onClick,
+            }: {
+              onClick: (ev: React.MouseEvent<HTMLElement>) => void;
+            }) => <Portrait onClick={onClick} />}
+          />
+        </Box>
+      );
+      const popoverContainerContentElement = popoverContainerContent(page);
+
+      await expect(popoverContainerContentElement).toHaveCSS(
+        "position",
+        "absolute"
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "top",
+        top
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "right",
+        right
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "bottom",
+        bottom
+      );
+
+      await assertCssValueIsApproximately(
+        popoverContainerContentElement,
+        "left",
+        left
+      );
+    });
+  });
+
+  test("should render with ariaDescribedBy", async ({ mount, page }) => {
+    await mount(<PopoverContainerComponent ariaDescribedBy={testString} />);
+
+    await expect(popoverContainerContent(page)).toHaveAttribute(
+      "aria-describedby",
+      testString
+    );
+  });
+
+  test("should render with openButtonAriaLabel", async ({ mount, page }) => {
+    await mount(<PopoverContainerComponent openButtonAriaLabel={testString} />);
+
+    const popoverContainerOpenComponent = getDataElementByValue(
```

Comment:
**suggestion** (non-blocking): I'm a little curious why you haven't just used `popoverSettingsIcon` here (btw that seems a strange name for what is the icon to open it...), and `popoverCloseIcon` in the following test, since you've bothered defining them and have even imported them to use in other tests

Reply:
Good point. I'll make these follow the same pattern of using the settings icon rather than the data element value.